eng/pipelines/test-integration-job.yml

```diff
@@ -32,6 +32,7 @@ steps:
       -configuration ${{ parameters.configuration }}
       -msbuildEngine dotnet
       -prepareMachine
+      -sign
```

Comment:
Not sure if this was actually required. It's part of the integration test jobs that get mirrored to DevDiv. The other portion of the cibuild signed so I made this one too in order to reflect the top-level azure-pipelines.yml

Reply:
If this is just for test, then it probably doesn't need to be signed.

Reply:
We specifically should not do this because it's a test build which should not ship (or really even leave the machine.